.github/workflows/checks.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: SIT Environment CI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run yamllint check
+        run: make yamllint
```

Comment:
Can we have the above change in a separate commit? Probably after another individual commit which adds the `yamllint` Makefile target?

Reply:
Sure. I'll split the commits in the next update.